projects/ama-sdk/src/lib/code-editor/components/code-editor/code-editor.component.scss

```diff
@@ -1,5 +1,7 @@
 @mixin amasdk-code-editor($theme) {
-    // $primary: map-get($theme, danger);
+    .mat-dialog-container .mat-tab-body.mat-tab-body-active {
```

Comment:
As I see, the problem with this approach is that we redefine globally this selector. What we should have for every different dialog containers to have a dedicated selector, which we can set css rules to.